docs/additional-functionality/rapids-udfs.md

```diff
@@ -52,7 +52,7 @@ Other forms of Spark UDFs are not supported, such as:
 
 For supported UDFs, the RAPIDS Accelerator will detect a GPU implementation
 if the UDF class implements the
-[RapidsUDF](../../sql-plugin-api/src/main/java/com/nvidia/spark/RapidsUDF.java)
+[RapidsUDF](https://github.com/NVIDIA/spark-rapids/blob/main/sql-plugin-api/src/main/java/com/nvidia/spark/RapidsUDF.java)
```

Comment:
LGTM  with the nit

We haven't been following this much but best practice for github seems to link to git blob permalinks https://github.com/NVIDIA/spark-rapids/blob/3d72579dfb247bc1569dc5ec17d9b1a6a8fced19/sql-plugin-api/src/main/java/com/nvidia/spark/RapidsUDF.java

technically branch can be renamed / deleted making the link obsolete.

Reply:
I thought about that, but the problem with a permalink is that you have to remember to update all of them when you make a change to the file. I could go either way though, but I think it's outside the scope of this for now.